azure-pipelines-ios.yaml

```diff
@@ -40,8 +40,13 @@ steps:
   - task: InstallAppleProvisioningProfile@1
     displayName: 'Install an Apple provisioning profile for Distribution'
```

Comment:
good job! I guess that's the advantage of having it commited in yaml.

Reply:
It is! :)